src/notifiers/apprise.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import apprise
+
+from models import Config, Item
+from models.errors import AppriseConfigurationError, MaskConfigurationError
+from notifiers import Notifier
+
+log = logging.getLogger('tgtg')
+
+
+class Apprise(Notifier):
+    """
+    Notifier for Apprise. \n
+    For more information on Apprise visit\n
+    https://github.com/caronc/apprise
+    """
+
+    def __init__(self, config: Config):
+        self.enabled = config.apprise.get("enabled", False)
+        self.body = config.apprise.get("body")
+        self.url = config.apprise.get("url")
+        self.timeout = config.apprise.get("timeout", 60)
```

Comment:
I have removed `apprise.timeout` because Apprise has two timeout values - connect and read - and they can be configured via `url` as [global parameters](https://github.com/caronc/apprise/wiki/URLBasics).